parser.c

```diff
@@ -478,16 +479,46 @@ static struct kanshi_profile *parse_profile(struct kanshi_parser *parser) {
 	}
 }
 
-static struct kanshi_config *_parse_config(struct kanshi_parser *parser) {
-	struct kanshi_config *config = calloc(1, sizeof(*config));
-	wl_list_init(&config->profiles);
+static bool parse_config_file(const char *path, struct kanshi_config *config);
+
+static bool parse_include_command(struct kanshi_parser *parser, struct kanshi_config *config) {
+	// Skip the 'include' directive.
+	if (!parser_expect_token(parser, KANSHI_TOKEN_STR)) {
+		return false;
+	}
+
+	if (!parser_read_line(parser)) {
+		return false;
+	}
+
+	if (parser->tok_str_len <= 0) {
+		return true;
+	}
+
+	wordexp_t p;
+	if (wordexp(parser->tok_str, &p, WRDE_SHOWERR | WRDE_UNDEF) != 0) {
+		fprintf(stderr, "Could not expand include path: '%s'\n", parser->tok_str);
+		return false;
+	}
+
+	char **w = p.we_wordv;
+	for (size_t idx = 0; idx < p.we_wordc; idx++) {
+		if (!parse_config_file(w[idx], config)) {
+			fprintf(stderr, "Could not parse included config: '%s'\n", w[idx]);
+			return false;
+		}
+	}
+	wordfree(&p);
+	return true;
+}
 
+static int _parse_config(struct kanshi_parser *parser, struct kanshi_config *config) {
```

Comment:
The return value should be updated to `bool` here.

Reply:
Ah, sorry, missed this in my late-night coding session. :|